compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangNodeTransformer.java

```diff
@@ -4601,6 +4611,17 @@ private BLangIdentifier createIdentifier(Location pos, String value, Set<Whitesp
         return bLIdentifer;
     }
 
+    private BLangLiteral createEmptyStringLiteral(TemplateExpressionNode expressionNode) {
```

Comment:
shall we directly pass the position, it feels weired to send in an expression to a function that create empty string literal.